### PR TITLE
Capstone_m2: Feature/structure

### PR DESCRIPTION
**Pull request by mateo951**
- Added project webpack set-up
- Added nav markup
- Added nav styling
- Added style.css
- Added .DS_Store to gitignore
- Added linters
- Added fotter to homepage
- Added basic structure for API functionality
- Added comments popup markup
- Added comments popup styling
- Added comment.js file and add comment Dom manipulations and evenlistener
- Added .vscode/ to .gitignore
- Added display of items
- Added conflict solving

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup ESLint
+        run: |
+          npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+          [ -f .eslintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.eslintrc.json
+      - name: ESLint Report
+        run: npx eslint .
```
Comment (fortuneonyeka) [inline .github/workflows/linters.yml:62]: Nice gitflow, however, you need to fix the linter errors

**Review by fortuneonyeka — CHANGES_REQUESTED** [review]
Hello @mateo951 , you have an amazing job here. Very good use of ES6 syntax.
However you do need to fix the linter errors.

Good job and happy coding👍👍👍

File: .github/workflows/linters.yml (hunk 1 above)
Comment (mateo951) [reply .github/workflows/linters.yml:62]: Done ✔ Thanks

**Comment by mateo951** [comment]
STATUS: CHANGES REQUESTED 🛑🛑
Hello @fortuneonyeka 👋🏽
You have done a very great job so far 👏🏽👏🏽.

Awesome job @:

Nice setup for webpack!
You don't have any CSS redundancy issues. Awesome!
JavaScript best practices are also good!
Nice work when manipulating the DOM and using ES6 syntax!

Required Changes ♻️
-  For better semantics, Its recommended using <nav> on navbars 
https://github.com/mateo951/js_capstone_m2/blob/268cd943d289c1b16ffdf8f64c1b7e61433df570/src/index.html#L11-L17

See inline comments for further explanation
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Review by fortuneonyeka — APPROVED** [review]

**Pull request merged**